{"uid":"4b4757e66a1912dae1a509f688f20b0f","name":"categories","children":[{"name":"Product defects","children":[{"name":"snappi_ixnetwork.exceptions.SnappiIxnException:   File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/snappi_api.py\", line 584, in get_metrics\n    response = self.protocol_metrics.results(request)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/protocolmetrics.py\", line 445, in results\n    return self._filter_stats(protocol)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/protocolmetrics.py\", line 418, in _filter_stats\n    return self._get_per_device_group_stats(protocol)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/protocolmetrics.py\", line 314, in _get_per_device_group_stats\n    raise Exception(msg)\n \n                Could not retrive drill down view                 at row index 0 Failed to get the View.href property as the View object has no encapsulated resources. Either the View.find() method was not called or it failed to retrieve any resources from the server.","children":[{"name":"test_encrypt_with_mka","uid":"6e6eecc4b75e5ac1","parentUid":"c30c85a6c46c7e04bf2115be7873f128","status":"failed","time":{"start":1741757585447,"stop":1741757681721,"duration":96274},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]},{"name":"test_encrypt","uid":"6fc049fc58fc2df3","parentUid":"c30c85a6c46c7e04bf2115be7873f128","status":"failed","time":{"start":1741757585447,"stop":1741757638100,"duration":52653},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"c30c85a6c46c7e04bf2115be7873f128"},{"name":"snappi_ixnetwork.exceptions.SnappiIxnException:   File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/snappi_api.py\", line 450, in set_control_state\n    self.traffic_item.transmit(request_payload)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/trafficitem.py\", line 1307, in transmit\n    self._api._traffic.Apply()\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/testplatform/sessions/ixnetwork/traffic/traffic.py\", line 1395, in Apply\n    return self._execute(\"apply\", payload=payload, response_object=None)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/base.py\", line 452, in _execute\n    response = self._connection._execute(url, payload)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 289, in _execute\n    return self._send_recv(\"POST\", url, payload)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 572, in _send_recv\n    return self._poll()\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 640, in _poll\n    return self._process_response_status_code(\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 476, in _process_response_status_code\n    raise BadRequestError(message, response.status_code)\n  Error in L2/L3 Traffic Apply","children":[{"name":"test_vxlan_b2b","uid":"f91ab755754a9cfd","parentUid":"ef6804d4159127be406a6111d9b6a662","status":"failed","time":{"start":1741757585447,"stop":1741757733893,"duration":148446},"flaky":true,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"ef6804d4159127be406a6111d9b6a662"},{"name":"Exception: Time out occurred while waiting for stats to be as expected","children":[{"name":"test_mka_stats","uid":"b2337e82a0dfa901","parentUid":"3596725c47ed555d65da3fbf3423e1cd","status":"failed","time":{"start":1741757585447,"stop":1741757735061,"duration":149614},"flaky":true,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"3596725c47ed555d65da3fbf3423e1cd"},{"name":"assert 0 == 1\n +  where 0 = len(<ixnetwork_restpy.testplatform.sessions.ixnetwork.availablehardware.chassis.chassis.Chassis object at 0x7f1d1c88e540>)","children":[{"name":"test_counter_ip_dscp","uid":"93222e09a5555562","parentUid":"36adf6ff29f829604034c80d02d614be","status":"failed","time":{"start":1741757585447,"stop":1741757610577,"duration":25130},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"36adf6ff29f829604034c80d02d614be"}],"uid":"8fb3a91ba5aaf9de24cc8a92edc82b5d"}]}